device-types/Fujitsu/primergy-rx2540-m5.yml

```diff
@@ -0,0 +1,34 @@
+---
+manufacturer: Fujitsu
+model: PRIMERGY RX2540 M5
+slug: fujitsu-primergy-rx2540-m5
+u_height: 2
+airflow: front-to-rear
+is_full_depth: true
+weight: 25
+weight_unit: kg
+comments: '[Fujitsu Server PRIMERGY RX2540 M5 Rack Server](https://www.fujitsu.com/vn/en/imagesgig5/Data_Sheet_FUJITSU_Server_PRIMERGY_RX2540_M5_Rack_Server.pdf)'
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+    maximum_draw: 715
+  - name: Power 2
+    type: iec-60320-c14
+    maximum_draw: 715
```

Comment:
It looks like these PSUs might be modular. If this is the case, the proper way to model this is to add them under `module-bays` and then create the associated modular PSUs under the module-types folder

Reply:
Converted to module-bays